package.json

```diff
@@ -8,7 +8,7 @@
   "license": "MIT",
   "scripts": {
     "test": "yarn hardhat test",
-    "build": "yarn hardhat compile",
+    "build": "yarn hardhat compile && yarn tsc",
```

Comment:
I think we should change to `yarn hardhat compile && yarn hardhat typechain && yarn tsc` just to be safe.

If you just run `yarn hardhat compile`, but hardhat decides it doesn't need to recompile, then the `typechain-types` directory is not created